samples/ballerina-by-example/examples/salesforce-client-connector/salesforcerest-client-connector.bal

```diff
@@ -0,0 +1,63 @@
+import org.wso2.ballerina.connectors.salesforcerest;
+
+import ballerina.lang.jsons;
+import ballerina.lang.system;
+import ballerina.net.http;
+
+function main (string[] args) {
+
+    http:Response response;
+    json JSONResponse;
+
+    string accessToken = args[0];
+    string clientID = args[1];
+    string clientSecret = args[2];
+    string refreshToken = args[3];
+    string apiInstance = args[4];
+    string refreshEndpoint = args[5];
+    string apiVersion = "v32.0";
+
+    salesforcerest:ClientConnector salesforceClient;
+    // Create a Salesforcerest Client Connector with the arguments accessToken, clientID, clientSecret, refreshToken, apiInstance, refreshEndpoint
+    salesforceClient = create salesforcerest:ClientConnector(accessToken, clientID, clientSecret, refreshToken,
+                                                             apiInstance, refreshEndpoint);
+
```

Comment:
Can you please make sure all the lines are not more than 90 columns length. So it iwill appear nicely in the generated html page. Please recheck all the code and comment lines